packages/python-odfpy/python-odfpy.spec

```diff
@@ -3,10 +3,10 @@
 
 Name:           python-%{pypi_name}
 Version:        1.4.1
-Release:        1%{?dist}
+Release:        2%{?dist}
 Summary:        Python API and tools to manipulate OpenDocument files
 
-License:        None
+License:        GPL-2.0-or-later or Apache-2.0
```

Comment:
I think the `-or-later` isn't really Fedora type. Shouldn't it be `GPL-2+`?

Reply:
blah blah blah, can't we have a standard for writing license identifiers down? ;)

Reply:
No, the world doesn't work that way. That would be too easy.